src/Columns/ColumnVector.h

```diff
@@ -136,7 +136,9 @@ class ColumnVector final : public COWHelper<ColumnVectorHelper, ColumnVector<T>>
 
     /// Sugar constructor.
     ColumnVector(std::initializer_list<T> il) : data{il} {}
-
+    #ifdef __SSE4_2__
+    ColumnPtr replicateSSE2(const IColumn::Offsets & offsets) const;
```

Comment:
It is strange that the function is named 'SSE2' while it is using SSE 4.2 and looks like SSSE3 is enough.

Reply:
ok i will change it.